### DISCUSSION TITLE
Enable/Disabled Javascript,remote fonts etc. by keyboard

### DISCUSSION THREAD

**Discussion by critkitten** [uBlockOrigin/uBlock-issues#2402] [Ideas] [answered]
Would it be possible to enable/disable such feature with keyboard combination?

**Comment by gothic-bum** [comment, marked as answer]
See "Relax blocking mode" keyboard shortcut for Javascript:  
https://github.com/gorhill/uBlock/wiki/Keyboard-shortcuts
https://github.com/gorhill/uBlock/wiki/Advanced-settings#blockingprofiles
